data/deprecated.json

```diff
@@ -227,6 +227,10 @@
     "old": {"barrier": "wood_fence"},
     "replace": {"barrier": "fence", "fence_type": "wood"}
   },
+  {
+    "old": {"bench": "capacity"},
```

Comment:
The old tags for this would be `amenity=bench` + `capacity=*`. The replacement tags would then be `amenity=bench` + `seats=$1`.